packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -1573,6 +1573,33 @@ describeWithDOM('mount', () => {
         });
       });
     });
+
+    describeIf(is('>= 16.8'), 'hooks', () => {
+      it('handles useState', () => {
+        const ComponentUsingStateHook = () => {
+          const [count] = React.useState(0);
+          return <div>{count}</div>;
+        };
+
+        const wrapper = mount(<ComponentUsingStateHook />);
+
+        expect(wrapper.find('div').length).to.equal(1);
+        expect(wrapper.find('div').text()).to.equal('0');
+      });
+      it('handles setState returned from useState', () => {
```

Comment:
```suggestion

      it('handles setState returned from useState', () => {
```

Reply:
@ljharb Can we add setprops test case to simulate componentwillReceiveProps and test using act ?

Reply:
sure, that sounds great.